examples/cms-tina/.tina/schema.ts

```diff
@@ -0,0 +1,108 @@
+
+import { defineSchema, defineConfig } from "tinacms";
```

Comment:
Could we run Prettier over this for formatting?

Reply:
I ran prettier no changes were made 🤷

Reply:
Hmmm. I noticed the empty line at the top of the line. Maybe it didn't pick that up?

Reply:
Apparently not,I will fix it on the way through